operator-framework/src/test/java/io/javaoperatorsdk/operator/baseapi/builtinresourcecleaner/BuiltInResourceCleanerIT.java

```diff
@@ -8,13 +8,21 @@
 import org.slf4j.LoggerFactory;
 
 import io.fabric8.kubernetes.api.model.Service;
+import io.javaoperatorsdk.annotation.Sample;
 import io.javaoperatorsdk.operator.ReconcilerUtils;
 import io.javaoperatorsdk.operator.dependent.standalonedependent.StandaloneDependentResourceIT;
 import io.javaoperatorsdk.operator.junit.LocallyRunOperatorExtension;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
 
+@Sample(
+    tldr = "Cleanup handler for built-in Kubernetes resources",
+    description =
+        "Demonstrates how to implement cleanup handlers (finalizers) for built-in Kubernetes"
+            + " resources like Service and Pod. These resources don't use generation the same way"
+            + " as custom resources, so this sample shows the proper approach to handle their"
+            + " lifecycle and cleanup logic.")
```

Comment:
We are past JDK 17 so I would rewrite these all to """


```suggestion
    description = """
        Demonstrates how to implement cleanup handlers (finalizers) for built-in Kubernetes
        resources like Service and Pod. These resources don't use generation the same way
        as custom resources, so this sample shows the proper approach to handle their
        lifecycle and cleanup logic.""")
```